examples/extraResources/README.md

```diff
@@ -0,0 +1,29 @@
+# extraResources
+
+Kedge might not support all the things that Kubernetes has, but kedge would not
+come in your way to define anything that Kubernetes understands.
+
+For e.g. right now there is no way to define secrets in kedge, but you can still
```

Comment:
I think we can't remove those lines, as there is already is PR for secrets.

Some alpha features like CronJob might be better example